prov/efa/src/efa_base_ep.c

```diff
@@ -586,11 +586,12 @@ bool efa_qp_support_op_in_order_aligned_128_bytes(struct efa_qp *qp, enum ibv_wr
  * If the base_ep is not binded to an EQ, or write to EQ failed,
  * this function will print the error message to console, then abort()
  *
- * @param[in,out] ep	base endpoint
- * @param[in] err	OFI error code
+ * @param[in,out] ep		base endpoint
+ * @param[in] err			OFI error code
  * @param[in] prov_errno	provider error code
+ * @param[in] should_abort 	If true, aborts when no EQ is available
  */
-void efa_base_ep_write_eq_error(struct efa_base_ep *ep, ssize_t err, ssize_t prov_errno)
```

Comment:
I do not think we need this commit at all, we shouldn't even call this function when the tx error can be ignored